test/extensions/clusters/eds/eds_test.cc

```diff
@@ -1403,6 +1403,133 @@ TEST_F(EdsTest, EndpointMovedToNewPriority) {
     EXPECT_FALSE(hosts[1]->healthFlagGet(Host::HealthFlag::FAILED_ACTIVE_HC));
     EXPECT_FALSE(hosts[1]->healthFlagGet(Host::HealthFlag::PENDING_DYNAMIC_REMOVAL));
   }
+
+  // 1 -> 2
+  // Moves all the endpoints to priority 1.
```

Comment:
priority 2 right?

Reply:
yes you're right. I'll just delete this line since I don't mention it below.